client/signup/steps/domains/index.jsx

```diff
@@ -18,7 +18,8 @@ var StepWrapper = require( 'signup/step-wrapper' ),
 	Notice = require( 'components/notice' ),
 	abtest = require( 'lib/abtest' ).abtest,
 	signupUtils = require( 'signup/utils' ),
-	i18n = require( 'lib/mixins/i18n' );
+	i18n = require( 'lib/mixins/i18n' ),
```

Comment:
Thanks for the tip! The library was already included above, so I removed that reference.